airflow/providers/google/cloud/example_dags/example_mssql_to_gcs.py

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import os
+
+from airflow import models
+from airflow.providers.google.cloud.transfers.mssql_to_gcs import MSSQLToGCSOperator
+from airflow.utils import dates
+
+GCS_BUCKET = os.environ.get("GCP_GCS_BUCKET", "example-airflow")
+FILENAME = 'test_file'
+
+SQL_QUERY = "USE airflow SELECT * FROM Country;"
+
+with models.DAG(
+    'example_mssql_to_gcs',
+    default_args=dict(start_date=dates.days_ago(1)),
```

Comment:
```suggestion
    start_date=None,
```

I believe we are moving away from unnecessary `default_args` and dynamic `start_data`.

Reply:
Yes and we've been adding `catchup=False` in all example DAGs to ward off DagRun headaches for new users.

@uranusjr I could be wrong here but having `start_date=None` would throw an exception that the `start_date` parameter is missing right?

Reply:
Uh, yes you’re right. Don’t do `start_date=None`.

Reply:
@Bowrna Any valid, static `start_date` value is fine. Also, @uranusjr's suggested change of not having the `start_date` in `default_args` is a good one!